src/urllib3/poolmanager.py

```diff
@@ -383,6 +408,17 @@ class ProxyManager(PoolManager):
         HTTPS/CONNECT case they are sent only once. Could be used for proxy
         authentication.
 
+    :param _enable_https_proxies:
```

Comment:
I'm not a fan of adding flags, especially if this is a new feature. Can we have this feature be additive and then do what we're already doing in this PR and error out in an invalid scenario (HTTPS+HTTPS)?

Logical next-steps are to figure out if we can also support HTTPS+HTTPS before we release this so there's less confusion.

Reply:
Sounds good, I'll error out then. I preferred to set a warning as I was concerned that multiple clients with an incorrect configuration (i.e HTTPS + HTTPS) would be affected. But then again, it's an incorrect configuration so they might as well fix it now. 

That way, when we add the HTTPS + HTTPS support clients should have their configuration up to date.

I'll make the changes and update here.

Reply:
That sounds great, thank you for this work! It'll make many more users connections secure. :)